Movies/Models/MovieModel.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  MovieModel.swift
+//  Movies
+//
+//  Created by Tyler Green on 5/12/23.
+//
+
+import Foundation
+
+struct MovieModel: Codable {
```

Comment:
So the json response for the `search` endpoint has a key `results` which maps to an array of json objects (the movies). For codable to work you will need another object that represents the outer object. So something like 
```
struct SearchResults: Codable {
    let results: [MovieModel]
}
```

When you make the network call the `SearchResults` model is what you decode from the response. Then you can just return the array of movie models to the UI layer.